src/routes/accomodation.routes.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import multer from "multer";
+import { createAccomodation, getAllAccomodation,deleteAccomodation, updateAccomodation,getSingleAccomodation} from "../services/accomodation.service.js";
+const router=express.Router();
+const storage=multer.diskStorage({});
+const fileFilter=(req,file,cb)=>{
+    if(file.mimetype.startsWith("image"))
+    {
+cb(null,true);
+    }
+    else{
+        cb("invalid image file!",false);
+    }
+};
+const uploads=multer({storage,fileFilter});
+router.get('/', getAllAccomodation);
+router.post('/create',uploads.single("image"),createAccomodation);
+router.put('/update/:id',uploads.single("image"),updateAccomodation);
+router.delete('/delete/:id',deleteAccomodation);
+router.get('/:id',getSingleAccomodation)
```

Comment:
we have changed the structure of where routers are supposed to be. also rename `routes` folder to `routers`